### ISSUE TITLE
Can't reproduce the results.

### ISSUE THREAD

**Issue by iioSnail** [jiahaozhenbang/SCOPE#7]
I try to reproduce the result in the paper. I completely follow the environment in the `README.md` file. 

I have run 30 epochs by 'train.sh' script, and the final checkpoints are the following:

```
epoch=23-df=79.3537-cf=78.0969.ckpt 
epoch=25-df=80.1070-cf=78.1445.ckpt 
epoch=26-df=80.1810-cf=78.5520.ckpt 
epoch=28-df=80.1802-cf=78.7387.ckpt  
epoch=29-df=80.2158-cf=78.5971.ckpt
```

I used the final checkpoint to evaluate the model by sighan 2015 and I got the results is following:

```
# without CIC
'sent-detect-acc': 85.36363636363636, 'sent-detect-p': 78.10858143607706, 'sent-detect-r': 82.43992606284658, 'sent-detect-f1': 80.21582733812951, 'sent-correct-acc': 84.54545454545455, 'sent-correct-p': 76.5323992994746, 'sent-correct-r': 80.77634011090574, 'sent-correct-f1': 78.59712230215827, 'char-detect-f1': 86.45614035087719, 'char-correct-f1': 91.12964366944655

# with CIC
{'sent-detect-acc': 86.27272727272727, 'sent-detect-p': 79.75133214920072, 'sent-detect-r': 82.99445471349352, 'sent-detect-f1': 81.34057971014492, 'sent-correct-acc': 85.45454545454545, 'sent-correct-p': 78.15275310834814, 'sent-correct-r': 81.33086876155268, 'sent-correct-f1': 79.71014492753623, 'char-detect-f1': 87.1578947368421, 'char-correct-f1': 91.38972809667673}
```

The sentence-level correction results of the `without CIC` is the same as the Ablation Study. But the other results are totally different compared to the results in the paper.

I want to know what I did wrong and how to get the results in the paper. Looking forward to your reply.

**Comment by jiahaozhenbang** [comment]
We also found this problem in the replication. Due to random.shuffle() used in [preprocessing](https://github.com/jiahaozhenbang/SCOPE/blob/main/data_process/get_train_data.py#L133) without a fixed seed, the training data may not be exactly the same as those in our original experiments.
